mrs/task_allocation/bidder.py

```diff
@@ -44,12 +44,15 @@ def __init__(self, robot_id, stp_solver, bidding_rule, auctioneer_name, **kwargs
         self.auctioneer_name = auctioneer_name
         self.bid_placed = None
 
+        self.experiment = kwargs.get('experiment')
```

Comment:
How does this affect what is being run in `ropod-fms`? Is this name the right one?